### ISSUE TITLE
Error decoding packet unknown with contents

### ISSUE THREAD

**Issue by PiiSaba** [PaperMC/Travertine#114]
[23:36:18 ERROR]: [/IP|YouTube0099] <-> DownstreamBridge <-> [Hub-1] - encountered exception
io.netty.handler.codec.DecoderException: Error decoding packet unknown with contents:

	at net.md_5.bungee.protocol.MinecraftDecoder.decode(MinecraftDecoder.java:69) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:56099f3:67]
	at net.md_5.bungee.protocol.MinecraftDecoder.decode(MinecraftDecoder.java:13) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:56099f3:67]
	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:88) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:56099f3:67]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:56099f3:67]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:56099f3:67]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:56099f3:67]
	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:102) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:56099f3:67]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:56099f3:67]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:56099f3:67]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:56099f3:67]
	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:323) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:56099f3:67]
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:297) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:56099f3:67]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:56099f3:67]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:56099f3:67]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:56099f3:67]
	at io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:286) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:56099f3:67]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:56099f3:67]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:56099f3:67]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:56099f3:67]
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1434) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:56099f3:67]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:56099f3:67]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:56099f3:67]
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:965) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:56099f3:67]
	at io.netty.channel.epoll.AbstractEpollStreamChannel$EpollStreamUnsafe.epollInReady(AbstractEpollStreamChannel.java:799) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:56099f3:67]
	at io.netty.channel.epoll.EpollEventLoop.processReady(EpollEventLoop.java:421) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:56099f3:67]
	at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:321) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:56099f3:67]
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:897) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:56099f3:67]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_222]
Caused by: java.lang.IndexOutOfBoundsException: readerIndex(0) + length(1) exceeds writerIndex(0): UnpooledSlicedByteBuf(ridx: 0, widx: 0, cap: 0/0, unwrapped: PooledUnsafeDirectByteBuf(ridx: 2, widx: 2, cap: 288))
	at io.netty.buffer.AbstractByteBuf.checkReadableBytes0(AbstractByteBuf.java:1433) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:56099f3:67]
	at io.netty.buffer.AbstractByteBuf.readByte(AbstractByteBuf.java:726) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:56099f3:67]
	at net.md_5.bungee.protocol.DefinedPacket.readVarInt(DefinedPacket.java:108) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:56099f3:67]
	at net.md_5.bungee.protocol.DefinedPacket.readVarInt(DefinedPacket.java:98) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:56099f3:67]
	at net.md_5.bungee.protocol.MinecraftDecoder.decode(MinecraftDecoder.java:40) ~[Travertine.jar:git:Travertine-Bootstrap:1.13-SNAPSHOT:56099f3:67]
	... 27 more

**Comment by electronicboy** [comment]
Update

**Comment by PiiSaba** [comment]
I updated but console shows a error then the player got kicked.
Error message: <-> DownstreamBridge <-> [Hub-1] - bad packet ID, are mods in use!? Empty minecraft packet!

I can understand what the message said. But I don't know how to fix it.
Is there a way to fix it or avoid not to get kicked?

**Comment by mikroskeem** [comment]
See waterfall.yml

**Comment by electronicboy** [comment]
Looks like travertine was behind waterfall a build, the configuration option has been removed in favor of making that the standard behavior

**Comment by PiiSaba** [comment]
hmm
My main server is 1.8.8 spigot.
I am making a custom nick plugin so I made a sign gui like hypixel for getting user input for custom nickname.
I found this: https://www.spigotmc.org/threads/sign-gui-for-user-input.104394/page-2#post-1162152
and I used the API.

//
for(WrappedChatComponent chat : packet.getLines()) {
          Bukkit.getConsoleSender().sendMessage(StringEscapeUtils.unescapeJavaScript(chat.getJson()));
}
\\
About this section, If I send a messages, players got kicked and shows error messages that "bad packet ID, are mods in use!? Empty minecraft packet!" in console.
If I don't send messages, players don't kicked from the server.

I don't know why it is empty packet.
Is there any solutions?

*Sorry for the bad language, I'm not English speaker.
